spec/requests/statsd_middleware_spec.rb

```diff
@@ -1,63 +1,80 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'support/rx_client_helpers'
 require 'statsd_middleware'
 
 # TODO: possibly refactor this spec to be generic, not dependent on PrescriptionsController
 RSpec.describe StatsdMiddleware, type: :request do
```

Comment:
This is a really annoying spec, I wonder if we can follow the TODO here and refactor so it's generic instead of just using a different route